lightning/src/ln/msgs.rs

```diff
@@ -1167,16 +1172,25 @@ impl Writeable for Init {
 		// global_features gets the bottom 13 bits of our features, and local_features gets all of
 		// our relevant feature bits. This keeps us compatible with old nodes.
 		self.features.write_up_to_13(w)?;
-		self.features.write(w)
+		self.features.write(w)?;
+		encode_tlv_stream!(w, {
+			(3, self.remote_network_address, option)
+		});
+		Ok(())
 	}
 }
 
 impl Readable for Init {
 	fn read<R: Read>(r: &mut R) -> Result<Self, DecodeError> {
 		let global_features: InitFeatures = Readable::read(r)?;
 		let features: InitFeatures = Readable::read(r)?;
+		let mut remote_network_address: Option<NetAddress> = None;
+		decode_tlv_stream!(r, {
+			(3, remote_network_address, option)
```

Comment:
Could do `1` instead of `3` iiuc

Reply:
Nope, this is set by the spec, we can't control it.